src/main/java/uk/gov/pay/api/model/PaymentEventsResponse.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.pay.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import uk.gov.pay.api.model.links.PaymentLinksForEvents;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@ApiModel(value="PaymentEvents", description = "A List of Payment Events information")
+public class PaymentEventsResponse {
+    @JsonProperty("payment_id")
+    private final String paymentId;
+
+    private final List<PaymentEventResponse> events;
+
+    @JsonProperty("_links")
+    private PaymentLinksForEvents links = new PaymentLinksForEvents();
```

Comment:
Do we initialiser here ? Looks like it is always set in constructor

Reply:
Can I remove it in follow-up PR?